### DISCUSSION TITLE
setup.bat error, is it over for me?

### DISCUSSION THREAD

**Discussion by Ferrum38** [cyberofficial/Synthalingua#119] [Q&A] [answered]
I have gotten this error when using the setup.bat command after following the installation steps am I doing something wrong or will my OS have to be repaired? Thanks for any help!
![setup bat issue](https://github.com/user-attachments/assets/fca6b2cf-2203-4d15-85b7-da1a5dad4af0)

**Comment by cyberofficial** [comment]
Are you using powershell or command prompt? 

Powershell usually starts the line as PS {folderpath}>
If it is powershell, needs to be basic command prompt 
Did you also double click the setup file?

**Reply by Ferrum38** [reply]
Thank you! I have managed to run `setup.bat`, however now when running `livetranslation.bat` it has an error for No module named speech recognition. I have ran `pip install SpeechRecognition` and it seemed to download something but this didn't help with the problem, any Ideas? 
![module issue](https://github.com/user-attachments/assets/b0ea583e-8e4a-4152-be2b-c4f1f02fded3)

**Reply by cyberofficial** [reply]
The first error, of loading primary imports indicates that the environment isn't found. Is there any data folder in the same location?

**Reply by Ferrum38** [reply]
no folder named the same however all of the original files should be there. In imports.py it says that about half of the modules couldnt be imported properly.

**Reply by cyberofficial** [reply]
If there is no data folder then it failed to install the necessary files from Python. You'll need to make sure the command prompt window path is the same as the files location.

**Comment by cyberofficial** [comment, marked as answer]
Please try downloading the source files again, Or visit release page https://github.com/cyberofficial/Synthalingua/discussions/122 and download latest from itch.io

**Reply by Ferrum38** [reply]
yes, worked perfectly! thank you for all of your help and thank you for making such a great tool!

**Reply by cyberofficial** [reply]
> yes, worked perfectly! thank you for all of your help and thank you for making such a great tool!

If you need additional help, feel free to ask again or make a new issue/ticket. If it happens to you, greater chance it happens to others. So having it reported is greatly appreciated 😄 

You are welcome and thanks for the report 😊
